t/t7114-reset-sparse-checkout.sh

```diff
@@ -24,7 +24,7 @@ test_expect_success 'setup' '
 test_expect_success 'reset when there is a sparse-checkout' '
 	echo "/c" >.git/info/sparse-checkout &&
 	test_config core.sparsecheckout true &&
-	git checkout -b resetBranch &&
+	git checkout -B resetBranch &&
```

Comment:
Just pointing out this test change. This test script only exists in microsoft/git. The point of this change is for `git checkout -b` to not do the full reset, so changing this to `-B` forces the change.